src/main/java/uk/gov/hmcts/reform/professionalapi/controller/advice/ExceptionMapper.java

```diff
@@ -120,16 +112,8 @@ private static Throwable getRootException(Throwable exception) {
     }
 
     private ResponseEntity<Object> errorDetailsResponseEntity(Exception ex, HttpStatus httpStatus, String errorMsg) {
-
         LOG.error(HANDLING_EXCEPTION_TEMPLATE, ex.getMessage());
-        ErrorResponse errorDetails = ErrorResponse.builder()
-                .errorDescription(getRootException(ex).getLocalizedMessage())
-                .errorMessage(errorMsg)
-                .status(httpStatus).errorCode(httpStatus.value())
-                .timeStamp(getTimeStamp())
-                .build();
-
         return new ResponseEntity<>(
-                errorDetails, httpStatus);
+                new ErrorResponse(42, httpStatus, errorMsg, getRootException(ex).getLocalizedMessage(), getTimeStamp()), httpStatus);
```

Comment:
why we used 42 here? what is the use of it?

Reply:
Has been changed to include 404